test/attr/attr_concurrent.swift

```diff
@@ -118,3 +118,10 @@ func testCaseNonTrivialValue() {
 
   j = 17
 }
+
+func testExplicitConcurrentClosure() {
+  let fn = { @concurrent in
```

Comment:
Can you also add some tests that combine `@concurrent` with parameter names, `throws`, `async`, and/or a return type?

Reply:
Sure, will do.

Reply:
(It'll be a follow-up PR)